resolwe_bio/processes/go_enrichment/go.yml

```diff
@@ -9,11 +9,11 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/rnaseq:4.0.0
+        image: resolwebio/rnaseq:4.10.0
```

Comment:
This image still contains resdk=10.1.0. At some point it would make sense to bump it to the latest version. This is just a comment, things will work just fine as already are.

Reply:
Yes, noted! Resdk 7.0.0 that was included in the older image has issues now that ES was removed.